README.md

```diff
@@ -11,13 +11,13 @@ The markup tree provided by this package is comprised of immutable/persistent, t
 In your `Package.swift` Swift Package Manager manifest, add the following dependency to your `dependencies` argument:
 
 ```swift
-.package(url: "https://github.com/apple/swift-markdown.git", .branch("main")),
+.package(url: "https://github.com/apple/swift-markdown.git", branch: "main"),
 ```
 
 Add the dependency to any targets you've declared in your manifest:
 
 ```swift
-.target(name: "MyTarget", dependencies: ["Markdown"]),
+.product(name: "Markdown", package: "swift-markdown"),
```

Comment:
While i think it's important to mention that the module name and package name are different, i feel like this change doesn't tell the whole story. How about combining both the original and your change?

```suggestion
.target(name: "MyTarget", dependencies: [
    .product(name: "Markdown", package: "swift-markdown"),
]),
```

Reply:
Better, I agree